src/main/java/core/basesyntax/writer/FileWriterImpl.java

```
@@ -0,0 +1,16 @@
+package core.basesyntax.writer;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String report, String fileName) {
+        try {
+            Files.writeString(Paths.get(fileName), report);
+        } catch (IOException e) {
+            throw new RuntimeException("Error writing file " + fileName, e);
```

The catch block rethrows a RuntimeException which is acceptable per the checklist's guidance on not ignoring exceptions, but improve the error message to include the target path and mention whether the write failed due to IO (e.g., "Can't write report to file: " + fileName). The checklist states: "Don't ignore exceptions in try-catch blocks ... Let's rethrow a RuntimeException with an **informative** message and exception object." Make the message more explicit about the intent and include the original exception (you do include it already).